src/App.js

```diff
@@ -8,4 +8,4 @@ function App() {
   );
 }
 
-export default App;
+export default App;
```

Comment:
One thing that could be a good practice to adopt is adding a new, empty line at the end of your files. It's typically not a problem for .js files (though occasionally a problem can arise, especially if you are using anything that involves concatenation of your code), but is more important if you're working in something like python. Github diffs will catch it for you and denote it with this symbol.

Reply:
Thanks for the tip! I had actually been doing the opposite: removing empty lines at the end of my code as I thought it made things look cleaner. I just went through and added newlines to the end of my files.